lib/puppet/cloudpack/scripts/master_source_dev.erb

```diff
@@ -13,7 +13,7 @@ if [ -f /etc/redhat-release ]; then
 name=Puppet Labs PS (Unofficial)
 baseurl=http://yum.puppetlabs.com/el/\$releasever/dependencies/\$basearch
 enabled=1
-gpgkey=http://yum.puppetlabs.com/RPM-GPG-KEY-puppetlabs
+gpgkey=http://yum.puppetlabs.com/RPM-GPG-KEY-puppet
```

Comment:
I think you'll want to add the new key and not replace it here (and elsewhere), as older versions of puppet will still be signed with the old key.

```gpgkey=http://yum.puppetlabs.com/RPM-GPG-KEY-puppetlabs
                http://yum.puppetlabs.com/RPM-GPG-KEY-puppet```

^^ is the right syntax according to https://projects.puppetlabs.com/issues/8867.